ds3-autogen-c/src/test/resources/output/ComplexTypedefStruct.h

```diff
@@ -0,0 +1,7 @@
+typedef enum {
```

Comment:
this was intended as a comparison for test output, but it was more effective to do a line by line comparison with assert.contains()

I'll remove this whole folder